### PR TITLE
Improve grammar - 60-next-steps.Rmd

### PR DESCRIPTION
**Pull request by stavares843**
Improve grammar - 60-next-steps.Rmd

### CODE REVIEW

**Comment by stavares843** [comment]
https://github.com/carpentries-incubator/bioc-intro/issues/35

File: _episodes_rmd/60-next-steps.Rmd [hunk 1]
```diff
@@ -23,26 +23,26 @@ library("tidyverse")
 ```
 
 Data in bioinformatics is often complex.
-To deal with this, developers define specialised
+To deal with this, developers define specialized
```
Comment (lgatto) [inline _episodes_rmd/60-next-steps.Rmd:26]: This isn't a typo. We use British spelling in this lesson.

File: _episodes_rmd/60-next-steps.Rmd [hunk 2]
```diff
@@ -181,7 +181,7 @@ dim(rowData(se))
 ### Subsetting a SummarizedExperiment
 
 SummarizedExperiment can be subset just like with data frames,
-with numerics or with characters of logicals.
+with numerics or with characters of logic.
```
Comment (lgatto) [inline _episodes_rmd/60-next-steps.Rmd:184]: this should be logicals, as it refers to the R data type

**Review by lgatto — CHANGES_REQUESTED** [review]

File: _episodes_rmd/60-next-steps.Rmd (hunk 1 above)
Comment (stavares843) [reply _episodes_rmd/60-next-steps.Rmd]: done 🔨

File: _episodes_rmd/60-next-steps.Rmd [hunk 3]
```diff
@@ -23,26 +23,26 @@ library("tidyverse")
...
+[^Bioconductor]: The [Bioconductor](http://www.bioconductor.org) was initiated by Robert Gentleman, one of the two creators of the R language. Bioconductor provides tools dedicated to omics data analysis. Bioconductor uses the R statistical programming language and is open source and open development.
 
 
 To illustrate such an omics data container, we'll present the `SummarizedExperiment`
 class.
 
 ## SummarizedExperiment
 
-The figure below represents the anatomy of SummarizedExperiment.
+The figure below represents the anatomy of the SummarizedExperiment.
```
[the remark this answers is not in the document]
Comment (stavares843) [reply _episodes_rmd/60-next-steps.Rmd]: done 🔨

File: _episodes_rmd/60-next-steps.Rmd (hunk 2 above)
Comment (stavares843) [reply _episodes_rmd/60-next-steps.Rmd]: done 🔨

**Review by lgatto — APPROVED** [review]

**Comment by lgatto** [comment]
Thank you @stavares843 for the contribution!

**Pull request merged**